# moved to init.py due to this error:
# AttributeError: module 'models.db' has no attribute 'init_app'
